packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -245,7 +245,7 @@ export function _SignInStart(): JSX.Element {
   }
 
   const hasSocialOrWeb3Buttons = !!authenticatableSocialStrategies.length || !!web3FirstFactors.length;
-  const shouldAutofocus = !isMobileDevice() && hasSocialOrWeb3Buttons;
+  const shouldAutofocus = !isMobileDevice() && !hasSocialOrWeb3Buttons;
```

Comment:
❓ could we add a test about this condition? It seems that if there are social or web3 buttons we shouldn't autofocus on email input. right?